api4/config.go

```diff
@@ -320,6 +323,16 @@ func makeFilterConfigByPermission(accessType filterType) func(c *Context, struct
 			if tagValue == model.ConfigAccessTagCloudRestrictable {
 				continue
 			}
+			if tagValue == AnyTagValue {
```

Comment:
Lets add maybe a comment here describing what the any tag is. It took me a while to understand it.
```suggestion
                        // If a config setting has a access tag of any, it means that this partcular config setting will always be return to a system console admin no matter what
	if tagValue == AnyTagValue {
```
Not the best wording but an explanation of what the any tag is and its purpose would be nice for new developers. 
There is a blurb in config.go in model folder describing the permissions so maybe add an explanation there too? 
Off topic but I really think that blurb needs documentation outside of code.
![image](https://user-images.githubusercontent.com/17804942/112669510-2e3c1600-8e36-11eb-9b8c-6526ae4ea731.png)

Reply:
If you do decide to go the route of only checking read permissions, this could be simplified to
```suggestion
			if tagValue == AnyTagValue && hasPermissionsToAnySysconsoleRead(...) {
			     return true
```